src/safe_get.js

```diff
@@ -0,0 +1,18 @@
+function NonExistingPropertyException(object, property) {
+  this.object = object;
+  this.property = property;
+
+  this.toString = () => {
+    return `Property '${this.property}' is not found in object '${JSON.stringify(this.object)}'.`;
+  };
+};
+
+module.exports = () => {
```

Comment:
Exported function should remember whether it was already invoked to ensure it is invoked only once. It'll make require-safe-get-per-file policy much cheaper in terms of time & memory.